url.bs

```diff
@@ -1114,6 +1114,50 @@ refers to as well as its origin. It is important that these are cached as the <a
 be removed from the <a>blob URL store</a> between parsing and fetching, while fetching will still
 need to succeed.
 
+<div class=example id=example-url-components>
+ <p>The following table lists how <a>valid URL strings</a>, when <a lt="URL parser">parsed</a>, map
+ to a <a for=/>URL</a>'s components. Username and password are left as exercise for the reader.
```

Comment:
"left as an exercise for the reader" is a bit misleading because it implies that the reader might have to do some work. Instead I'd state this as something like

> Username and password are omitted from the table; in all the examples given they are the empty string. Similarly for blob URL entry, which is null in all the examples given.